code/style.css

```diff
@@ -1,7 +1,465 @@
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+  border: 1px solid #DB398D;
+  text-align: center;
+}
+
+.accordion a.active::after {
+  font-family: 'Ionicons';
+  content: '\f209';
+}
+
+.accordion .accordion-content {
+  padding: 0 18px;
+  background-color: white;
+  overflow: hidden;
+  display: none;
+  padding-block-start: 1em;
+  padding-block-end: 1em;
+  margin-block-end: 1em;
+  transition: all 0.2s ease 0.15s;
+}
+
+.accordion .accordion-content.active {
+  display: block;
+  transition: all 0.35s ease 0.15s;
+}
+
+/* form */
+
+.form-container {
+  background-color: #00c2db;
+  padding: 0 2em;
+  border: 1px solid #ccc  ;
+  color: #fff;
+}
+
+form {
+  /* Form outline */
+  width: 100%;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+form li + li {
+  margin-top: 1em;
+}
+
+
+label {
+  /* Uniform size & alignment */
+  display: inline-block;
+  font-size: 1.4rem;
+  text-align: left;
+  padding-bottom: 4px;
+}
+
+select,
+.text-input {
+  /* To make sure that all text fields have the same font settings
+     By default, textareas have a monospace font */
+  font: 1.2rem sans-serif;
+
+  /* Uniform text field size */
+  width: 100%;
+  height: 40px;
+  box-sizing: content-box;
+
+  /* Match form field borders */
+  border: 1px solid #999;
+}
+
+select,
+::placeholder {
+  /* Uniform box appearance */
+  color: grey;
+  padding: 2px 2px 2px 6px;
+}
+
+label.label-checkbox {
+  /* styling for the checkbox */
+  display: flex;
+  align-items: center;
+  font-size: 0.9rem;
+}
+
+.checkbox {
+  /* size of checkbox */
+  height: 20px;
+  width: 20px;
+}
+
+/* call to action-button styling */
+
+.btn-submit {
+  text-align: center;
+}
+
+button {
+  border: none;
+  background: #DB398D;
+  border-radius: 2em;
+  padding: 12px 20px;
+  color: #fff;
+  font-size: 1.4rem;
+  margin: 2.5em auto;
+}
+
+button:hover,
+button:focus {
+  color: #DB398D;
+  background: rgb(243, 237, 237);
+  outline: 0;
+  font-weight: bold;
+}
+
+/* footer */
+
+footer {
+  position: relative;
+  bottom: 0;
+  width: 100%;
+  background: #999;
+  text-align: center; 
+  color: #fff;
+  border-top: 1px solid #e0dfdf;
+  font-size: 0.8rem;
+  text-transform: uppercase;
+  padding-top: 25px;
+  padding-bottom: 25px;
+  z-index: 2;
+}
+
+/* general link styling for footer links */
+
+footer a:link {
+  color: cyan;
+}
+
+footer a:visited {
+  color: black;
+}
+
+footer a:active {
+  background-color: cyan;
+}
+
+/* hover effect on footer links */
+
+.footer-anchor, .footer-anchor > span {
+  position: relative;
+  color: inherit;
+  text-decoration: none;
+}
+
+.footer-anchor:before, .footer-anchor:after, .footer-anchor > span:before, .footer-anchor > span:after {
+  content: '';
+  position: absolute;
+  transition: transform .5s ease;
+}
+
+.footer-anchor:before {
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 2px;
+  background: #DB398D;
+  transform: scaleX(0);
+}
+
+.footer-anchor:hover:before {
+  transform: scaleX(1);
+}
```

Comment:
This is sooo cool, looks quite easy but adds so much to the page. Will try this out in a future assignment if that is ok with you ofc!